templates/expo-template-bare-minimum/ios/Podfile

```
@@ -31,7 +31,8 @@ target 'HelloWorld' do
     react_native_post_install(
       installer,
       config[:reactNativePath],
-      :mac_catalyst_enabled => false
+      :mac_catalyst_enabled => false,
+      :ccache_enabled => !!podfile_properties['apple.ccacheEnabled']
```

Oh wait, we convert all booleans to string when writing the JSON. So this needs to be this, right? (can be omitted, `false`, or `true`)

```suggestion
      :ccache_enabled => podfile_properties['apple.ccacheEnabled'] == 'true'
```

---

i didn't implement other generic types than string: https://github.com/expo/expo/blob/87efd0c9e667151ba26ca047a195d57ea9365273/packages/%40expo/config-plugins/src/utils/BuildProperties.types.ts#L28

the other reason is that on android, gradle.properties only supports strings.